esp/xslt/esdl2monitor.xslt

```diff
@@ -227,12 +232,12 @@ END;
 // we start to throttle, and maxRetries controls how many times inserts that fail because Cassandra is too busy
 // will be retried.
 
-monitorStoreRec getStoredMonitor(string id) := EMBED(cassandra : server(csndServer), user(csndUser), password(csndPassword), keyspace('eclresultmonitoring'))
-  SELECT monitorId, result from monitors WHERE monitorId=? LIMIT 1;
+monitorStoreRec getStoredMonitor(string id) := EMBED(cassandra : server(csndServer), user(csndUser), password(csndPassword), keyspace(csndKeySpaceFrom))
```

Comment:
Just pointing out the default for csnKeySpaceFrom is now 'monitors_a', not eclruseltmonitoring. Most likely by design but pointing out in case it's not.